app/hook/useDate.ts

```diff
@@ -0,0 +1,78 @@
+export default function useDate() {
+  type TypeMonth = {
+    1: string
+    2: string
+    3: string
+    4: string
+    5: string
+    6: string
+    7: string
+    8: string
+    9: string
+    10: string
+    11: string
+    12: string
+  }
+  const months: TypeMonth = {
+    1: "Enero",
+    2: "Febrero",
+    3: "Marzo",
+    4: "Abril",
+    5: "Mayo",
+    6: "Junio",
+    7: "Julio",
+    8: "Agosto",
+    9: "Septiembre",
+    10: "Octubre",
+    11: "Noviembre",
+    12: "Diciembre",
+  }
+
+  const getStringMonth = (number: any): string => {
+    const monthIndex: keyof TypeMonth = number
+    const monthString = months[monthIndex]
+    return monthString
+  }
+
+  const getYearMonthDayOfDate = (data: Date) => {
+    const MONTH_LAG = 1
+    const year = data.getFullYear()
+    const month = data.getMonth() + MONTH_LAG
+    const day = data.getDate()
+    return { year, month, day }
+  }
+
+  const transformDataToDataString = (data: Date): string => {
+    const { year, month, day } = getYearMonthDayOfDate(data)
+    const monthLetters = getStringMonth(month)
+    const dataString = `${day} ${monthLetters}, ${year}`
+    return dataString
+  }
+
+  type GenerateEndPointCardPostParam = {
```

Comment:
could we put this type near the other type? maybe after or before TypeMonth?
for organization